aclk/aclk_common.h

```diff
@@ -17,6 +47,8 @@ const char *aclk_proxy_type_to_s(ACLK_PROXY_TYPE *type);
 #define ACLK_PROXY_ENV "env"
 #define ACLK_PROXY_CONFIG_VAR "proxy"
 
+#define ACLK_MAX_QUERY_THREADS 4
```

Comment:
The query name setting code injects one character (thread number) to build a proper thread name. 

Any reason why not have a max default of 9 queries then?

Reply:
seem bit excessive but you are right it is just arbitrary limit. I can remove it if you think it is better

Reply:
Please remove it. We try to avoid arbitrary limits where possible. In this case - if the node is a parent with 100s of children then we may want the ACLK request processing to scale up to 10s or 100s of threads.

Reply:
OK I will remove the max limitation and related code